### PR TITLE
[1LP][RFR] Custom button: Service and Generic object test

### PR DESCRIPTION
**Pull request by digitronik**
Purpose or Intent
=================
- Service and Generic object custom button tests base
- display test for service and generic object
- Fix sometime `ssui` login problem with `wait`

{{pytest: cfme/tests/automate/custom_button/test_service_objects.py -vvv}}

### CODE REVIEW

File: cfme/base/ssui.py [hunk 1]
```diff
@@ -125,7 +125,6 @@ def address(self):
 @MiqImplementationContext.external_for(Server.login, ViaSSUI)
 def login(self, user=None, method=LOGIN_METHODS[-1]):
     if not user:
-        from cfme.configure.access_control import User
```
Comment (digitronik) [inline cfme/base/ssui.py]: Note: Imported but not used

File: cfme/services/myservice/ui.py [hunk 2]
```diff
@@ -339,6 +340,9 @@ class MyServiceAll(CFMENavigateStep):
     def step(self, *args, **kwargs):
         self.prerequisite_view.navigation.select('Services', 'My Services')
 
+    def resetter(self, *args, **kwargs):
+        self.view.myservice.tree.click_path('Active Services')
```
Comment (digitronik) [inline cfme/services/myservice/ui.py]: Note: reset view so it will maintain `All` view every time...

File: cfme/base/ssui.py [hunk 3]
```diff
@@ -145,7 +144,7 @@ def login(self, user=None, method=LOGIN_METHODS[-1]):
     time.sleep(3)
     login_view.flash.assert_no_error()
     self.browser.plugin.ensure_page_safe()
-    assert logged_in_view.is_displayed
+    logged_in_view.wait_displayed('10s')
```
Comment (sshveta) [inline cfme/base/ssui.py]: you can assert for this too .
assert logged_in_view.wait_displayed('10s')

Comment (digitronik) [reply cfme/base/ssui.py]: @sshveta I think, no need to assert... It will raise `Timeout` error. If fails to land within `10s` after a call.

**Pull request merged**